UnityProject/Assets/Scripts/Messages/Server/SoundMessages/PlaySoundMessage.cs

```diff
@@ -151,7 +151,8 @@ public override void Process()
 
 		public override string ToString()
 		{
-			return $"{nameof(SoundName)}: {SoundName}, {nameof(Position)}: {Position}, {nameof(Polyphonic)}: {Polyphonic}, {nameof(ShakeParameters)}: {ShakeParameters.ToString()}, {nameof(AudioSourceParameters)}: {AudioSourceParameters.ToString()}";
+			// null.ToString() will result in error and $"{null}}" will work fine, so I just removed ToString()
+			return $"{nameof(SoundName)}: {SoundName}, {nameof(Position)}: {Position}, {nameof(Polyphonic)}: {Polyphonic}, {nameof(ShakeParameters)}: {ShakeParameters}, {nameof(AudioSourceParameters)}: {AudioSourceParameters}";
 		}
```

Comment:
NullReferenceException: Object reference not set to an instance of an object
Assets.Scripts.Messages.Server.SoundMessages.PlaySoundMessage.ToString () (at /github/workspace/UnityProject/Assets/Scripts/Messages/Server/SoundMessages/PlaySoundMessage.cs:154)
Assets.Scripts.Messages.Server.SoundMessages.PlaySoundMessage.Process () (at /github/workspace/UnityProject/Assets/Scripts/Messages/Server/SoundMessages/PlaySoundMessage.cs:28)
GameMessageBase.Process (Mirror.NetworkConnection sentBy) (at /github/workspace/UnityProject/Assets/Scripts/Messages/GameMessageBase.cs:25)
GameMessageBase.PreProcess (Mirror.NetworkConnection sentBy, GameMessageBase b) (at /github/workspace/UnityProject/Assets/Scripts/Messages/GameMessageBase.cs:18)
Mirror.MessagePacker+<>c__DisplayClass6_0`2[T,C].<MessageHandler>b__0 (Mirror.NetworkConnection conn, Mirror.NetworkReader reader, System.Int32 channelId) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/MessagePacker.cs:146)
Mirror.NetworkConnection.InvokeHandler (System.Int32 msgType, Mirror.NetworkReader reader, System.Int32 channelId) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkConnection.cs:219)
Mirror.NetworkConnection.TransportReceive (System.ArraySegment`1[T] buffer, System.Int32 channelId) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkConnection.cs:272)
Mirror.ULocalConnectionToServer.Update () (at /github/workspace/UnityProject/Assets/Mirror/Runtime/LocalConnections.cs:110)
Mirror.NetworkClient.Update () (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkClient.cs:259)
Mirror.NetworkManager.LateUpdate () (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkManager.cs:255)